src/core/utils/url.js

```diff
@@ -23,11 +28,60 @@ export function removeUndefinedProps(object: Object): Object {
 export function addQueryParams(
   urlString: string,
   queryParams: { [key: string]: ?string | number } = {},
+  _config: typeof config = config,
 ): string {
+  let adjustedQueryParams = { ...queryParams };
+
+  if (
+    _config.get('enableFeatureUseUtmParams') &&
+    typeof queryParams.src !== 'undefined'
+  ) {
+    adjustedQueryParams = {
+      ...queryParams,
+      // Use UTM parameters instead of `src`, according to the PRD.
+      utm_source: DEFAULT_UTM_SOURCE,
+      utm_medium: DEFAULT_UTM_MEDIUM,
+      utm_content: queryParams.src,
+      src: undefined,
+    };
+  }
+
   const urlObj = url.parse(urlString, true);
   // Clear search, since query object will only be used if search property
   // doesn't exist.
   urlObj.search = undefined;
-  urlObj.query = removeUndefinedProps({ ...urlObj.query, ...queryParams });
+  urlObj.query = removeUndefinedProps({
+    ...urlObj.query,
+    ...adjustedQueryParams,
+  });
+
   return url.format(urlObj);
 }
+
+export function getQueryParametersForAttribution(
+  location: ReactRouterLocationType,
+  _config: typeof config = config,
+): Object {
+  let params = { src: location.query.src };
+
+  if (_config.get('enableFeatureUseUtmParams')) {
+    if (typeof location.query.src !== 'undefined') {
```

Comment:
When would this happen? If the flag is enabled, why would we have `src` on a URL?

Reply:
`src` is used on AMO to track attribution and it can be used by developers and/or Mozilla. It is passed from the query string to the addon manager in FF actually (for download stats). We are going to switch to UTM params instead of `src` and we won't need to pass those params to addon manager because the attribution data collection will be done by Firefox in a different way.

Reply:
I checked again based on your comment, which, I think, I misunderstood yesterday. This might not be needed after all because, while it makes things a bit easier in terms of code, it also adds a non-desired behavior.